docs/registry.md

```diff
@@ -0,0 +1,341 @@
+# Registry
+
+The registry is a place where applications metadata are stored and versioned. It can be used by a cozy 
+
+We define an application registry as an API. This should allow us to defer the real implementation of the registry storage and allow different store implementations.
+
+## Objects
+
+Two type of objects are managed in the registry: applications and versions.
+
+### Application
+
+An application object contains the following fields:
+
+- `name`: the application name
+- `type`: the application type ("webapp" or "konnector")
+- `editor`: the application editor name
+- `category`: the application category
+- `repository`: object with type and url of package repository
```

Comment:
I'm not sure to see what `repository` is. Is it https://github.com/cowy/cozy-photos-v3 for cozy-photos?